### PR TITLE
Deploy ICBC adapter to Openshift

### PR DESCRIPTION
**Pull request by pdittaro**
This PR is to deploy the ICBC Rest adapter to the Dev and Test environments.

### CODE REVIEW

File: openshift/.pipeline/Jenkinsfile [hunk 1]
```diff
@@ -146,19 +170,29 @@ pipeline {
...
+     },
+     'ICBCAdapter': {
+      sh "oc process -f openshift/templates/icbc-adapter.dc.yaml \
+         -p NAME=${ICBC_ADAPTER} \
+         -p APPLICATION_DOMAIN=${SUBDOMAIN}-test.${DOMAIN} \
+         -p MEMORY_LIMIT=${RUN_MEMORY} \
+         -p NAMESPACE=${TOOLS_NAMESPACE} \
+         -p TAG=test | oc apply -f - -n ${TEST_NAMESPACE}"
+      sh "oc tag ${ICBC_ADAPTER}:dev ${ICBC_ADAPTER}:test -n ${TOOLS_NAMESPACE}"
+      // WaitForDeploy(ICBC_ADAPTER, DEV_NAMESPACE)
```
Comment (akroon3r) [inline openshift/.pipeline/Jenkinsfile]: Was wondering why these comments are included if they're never used?

Comment (pdittaro) [reply openshift/.pipeline/Jenkinsfile]: Good point, we can uncomment these

File: openshift/.pipeline/Jenkinsfile [hunk 2]
```diff
@@ -85,6 +86,19 @@ pipeline {
          -p NAMESPACE=${TOOLS_NAMESPACE} | oc apply -f - -n ${TOOLS_NAMESPACE}"
       StartBuildAndWait("${SAMPLE_ADAPTER}-build", TOOLS_NAMESPACE)
       StartBuildAndWait("${SAMPLE_ADAPTER}-runtime", TOOLS_NAMESPACE)
+     },
+     'ICBCAdapter': {
```
Comment (alexjoybc) [inline openshift/.pipeline/Jenkinsfile]: Can we name this one IcbcRestAdapter:

image

Comment (pdittaro) [reply openshift/.pipeline/Jenkinsfile]: I have renamed the artifacts to reflect the title 'ICBC Rest Adapter'

Comment (alexjoybc) [reply openshift/.pipeline/Jenkinsfile]: Thanks @pdittaro

**Pull request merged**